resources/assets/components/InboxItem/index.js

```diff
@@ -30,11 +46,11 @@ class InboxItem extends React.Component {
           <h4>Why Statement</h4>
           <p>{post['signup']['why_participated']}</p>
           <ul className="form-actions -inline">
-            <li> <input className="button" value="Accepted"/></li>
+            <li> <input className="button" value="Accepted" onClick={this.onAcceptClick}/></li>
             <li> <input className="button -secondary" value="Rejected"/></li>
-            <br/>
-            <li> <input className="button -tertiary" value="Delete"/></li>
+            <li> <input className="button -tertiary" id="delete_post" value="Delete"/></li>
```

Comment:
I just added that so the text is left aligned instead of center just for some styling but can totally change back! Here's the difference:
without `delete_post` ID:
![screen shot 2017-03-31 at 4 32 39 pm](https://cloud.githubusercontent.com/assets/9019452/24568361/bac7417c-162f-11e7-8284-4515f93ffa55.png)
 with `delete_post` ID:
![screen shot 2017-03-31 at 4 32 13 pm](https://cloud.githubusercontent.com/assets/9019452/24568369/c24236aa-162f-11e7-9085-4fa2e554f46c.png)